components/brave_rewards/browser/publisher_info_database_unittest.cc

```diff
@@ -851,7 +852,7 @@ TEST_F(PublisherInfoDatabaseTest, GetActivityList) {
   filter_2.excluded = ledger::EXCLUDE_FILTER::FILTER_ALL;
   EXPECT_TRUE(publisher_info_database_->GetActivityList(0,
                                                         0,
-                                                        filter_2,
+                                                        filter_2.Clone(),
```

Comment:
`filter_2.Clone()` should use `std::move(filter_2)` as `filter_2` is no longer used

Reply:
fixed